packages/big-design/src/components/Button/__snapshots__/spec.tsx.snap

```diff
@@ -1,6 +1,58 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`render default button 1`] = `
+exports[`isLoading render loading button 1`] = `
```

Comment:
A bit of a mess with the snapshots, I'm not sure how to make it more "readable". I've reshuffled the tests slightly.